bin/fetch-configlet

```diff
@@ -23,27 +36,47 @@ def get_os():
 
 
 def get_arch():
-    return '64bit' if sys.maxsize > 2**32 else '32bit'
+    return "64bit" if sys.maxsize > 2 ** 32 else "32bit"
 
 
 def fetch_configlet():
-    latest = "https://api.github.com/repos/exercism/configlet/releases/latest"
-    resp = requests.get(latest)
-    data = resp.json()
+    with urlopen(
```

Comment:
Ok, we need to handle urllib.error.HTTPError being thrown here, probably via:

```python
from urllib.errors import HTTPError

[...]

try:
    with urlopen("https://api.github.com/repos/exercism/configlet/releases/latest") as request:
        status = request.status
        headers = dict(request.getheaders())
        response = request.read()
except HTTPError as err:
     status = err.code
     headers = dict(err.headers)
     response = err.reason
```

As right now it looks like a 403 is never actually reaching the `if status == 403` line.